source/grid/manifold_lib.cc

```diff
@@ -32,11 +32,13 @@ SphericalManifold<dim,spacedim>::SphericalManifold(const Point<spacedim> center)
 
 
 template <int dim, int spacedim>
-Point<spacedim>
+Tensor<1,spacedim>
 SphericalManifold<dim,spacedim>::get_periodicity()
 {
-  Point<spacedim> periodicity;
-  periodicity[spacedim-1] = 2*numbers::PI; // theta and phi period.
+  Tensor<1,spacedim> periodicity;
+  // In two dimensions, theta is periodic. In three dimensions the variable
+  // that is periodic is phi. We have no periodicity in theta in 3d.
+  periodicity[spacedim-1] = 2*numbers::PI;
```

Comment:
This will change soon. The implementation as it is right now is broken, and will need to be changed. I don't think we should use any more `ChartManifold` for `SphericalManifold` in 3D. The singularity at the poles is something we cannot deal with yet.